pebblo/app/storage/sqlite_db.py

```diff
@@ -104,6 +111,38 @@ def query_by_id(self, table_obj, id):
             )
             return False, err
 
+    @timeit
+    def query_by_list(
+        self,
+        table_obj: Type[DeclarativeMeta],
+        filter_key: str,
+        filter_values: List[str],
```

Comment:
I would recommend adding a `max_filter_values` to limit with a safe default value to avoid run-away memory usage. We can eventually add pagination support.

Reply:
Added.